macrel/output.py

```diff
@@ -0,0 +1,129 @@
+readme_output_abundance_mode = """
+--------------------------------------------------------------------------------------------------------------------------
+------------------------------------------          READ  ME        ------------------------------------------------------
```

Comment:
@celiosantosjr Use Markdown and spell it `README`

Reply:
Also, can you refactor this so that you have a common header that is always output + a variable part.

I have some improvements to the text, but can do them after merging.